test/mailers/previews/reservation_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+class ReservationMailerPreview < ActionMailer::Preview
+  def confirmation_email
+    ReservationMailer.confirmation_email(reservation: Reservation.first, 
```

Comment:
This will now work  where there is no reservation in the system, I would said that you  can use factory here just to build, without saving, a reservation for purpose of showing it

Reply:
Is it possible to use Factorybot in mailers previews ? I am having some trouble with that.